cli/command/registry.go

```diff
@@ -106,7 +106,12 @@ func ConfigureAuth(cli Cli, flUser, flPassword string, authconfig *registrytypes
 	if flUser = strings.TrimSpace(flUser); flUser == "" {
 		if isDefaultRegistry {
 			// if this is a default registry (docker hub), then display the following message.
-			fmt.Fprintln(cli.Out(), "Login with your Docker ID to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com to create one.")
+			fmt.Fprintln(cli.Out(), "Log in with your Docker ID or email address to push and pull images from Docker Hub. If you don't have a Docker ID, head over to https://hub.docker.com/ to create one.")
```

Comment:
maybe change "Docker ID" for "username" (for the first one) and "account" (for the second one), as I think `Docker ID` is a term that was used "in the past", but no longer in use.

```suggestion
			fmt.Fprintln(cli.Out(), "Log in with your username or email address to push and pull images from Docker Hub. If you don't have an account, head over to https://hub.docker.com/ to create one.")
```

Reply:
Docker ID is still the term we use, I think > https://docs.docker.com/docker-id/